.github/scripts/clean-clusters.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+if [ -z "${IS_CI}" ]; then
+  echo "Error: This script is intended to run only in CI environments."
+  echo "Running it in your account may have destructive effects."
```

Comment:
Nit: Maybe clarify that "destructive effects" means deleting clusters.

Reply:
Should be addressed by [da6531e](https://github.com/aws-samples/aurora-dsql-samples/commit/da6531ef2c0a994fbef5f257805b8ea9322de7f1)